src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -47,6 +47,10 @@
     public static final Prefix PREFIX_MAJOR = new Prefix("m/");
     public static final Prefix PREFIX_CAP = new Prefix("c/");
 
+    /* Sort keywords */
+    public static final Prefix PREFIX_SORT_ORDER = new Prefix("order/");
```

Comment:
suggestion: o/ for order and a/ for ascending?

Reply:
I think spelling it out in full will be clearer.